src/hooks/use-archive-site.ts

```diff
@@ -89,6 +89,13 @@ export function useArchiveSite() {
 				type: 'application/zip',
 			} );
 
+			const formData = [ [ 'import', file ] ];
+			const wordpressVersion = await getIpcApi().getWpVersion( siteId );
+			if ( wordpressVersion.length >= 3 ) {
+				// Minimum version length is '6.0'
```

Comment:
Nitpick: the condition is clear enough, so we don't need the comment

Reply:
Thanks for confirming it. I was not sure if it was clear enough.
I was thinking to make a comparison like, but I preferred to add it as a comment.
`if (  wordpressVersion.length >= 'x.y'.length ) {`

Anyway, if it's not clear for everybody, now they can read these comments 👍 

Removed the comment: https://github.com/Automattic/studio/pull/36/commits/d58a3ac28d01825e418dde78d75f312335089e3b